### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by jakob404** [victornpb/undiscord#277] [1. Questions & Answers]
Hello, recently I have benn trying to use the delete massages bot but on the bot responded with "Ended because API returned an empty page" . the bot works perfctly fine on all the other dms just on the one I realy want to delete it doesnt work
(screen shot bellow)
I ve trying to uninstal and instal again but it just doesnt work. i would apperciate if I would get an instruction.
Thank you 
![Posnetek zaslona 2021-11-26 102346](https://user-images.githubusercontent.com/72807566/143557816-14109c82-db07-4113-9cb4-88fdcae4dbe0.png)

**Comment by Toby222** [comment]
Seems like there's 25 system messages in the page?
try using the "before message with ID" filter to go to the last message from you

**Comment by KitsuTheFloof** [comment]
![image](https://user-images.githubusercontent.com/90358192/145628686-b31451b4-a4bf-4a0e-98df-4afcfbf1b351.png)
